subgraph/package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "bitsausage-subgraph",
+  "name": "bitsausage",
   "scripts": {
     "codegen": "graph codegen subgraph.yaml",
```

Comment:
`"codegen": "graph codegen --output-dir types/ subgraph.yaml",`

is the new command for codegen since we upgraded graph cli today